Template.php

```diff
@@ -1415,7 +1410,7 @@ protected function use_unnamed_params() {
       }
       if (preg_match("~\(?(1[89]\d\d|20\d\d)[.,;\)]*~", $dat, $match)) { #YYYY
         if ($this->blank('year')) {
-          $this->set('year', $match[1]);
+          $this->add('year', $match[1]);
```

Comment:
The difference between `add` and `set` is the reporting printed to the user; not clear that this change is necessary?